suite-native/module-authorize-device/src/navigation/PassphraseStackNavigator.tsx

```diff
@@ -40,14 +40,6 @@ const determineNativePassphraseFlowState = (
         return 'passphrase-checking-on-device';
     }
 
-    if (discovery.status === 'failed') {
-        return 'passphrase-redirecting';
-    }
-
-    if (discovery.status === 'complete') {
-        return 'passphrase-complete';
-    }
-
```

Comment:
these were never rendered but maybe it was intented render the "catch all route" for these cases?